packages/ui-kit-theme/src/overrides/MuiAutocomplete.tsx

```diff
@@ -0,0 +1,42 @@
+import { Theme } from '@mui/material';
+
+export default {
+  defaultProps: {
+    forcePopupIcon: false,
+  },
+  styleOverrides: {
+    paper: ({ theme }: { theme: Theme }) => {
+      const shadowColor = theme.palette.mode === 'light' ? '0' : '255';
+      return {
+        backgroundColor: theme.palette.mode === 'light' ? 'white' : 'black',
+        borderRadius: '12px',
+        marginTop: '10px',
+        boxShadow: `0px 4px 20px rgba(${(shadowColor + ',').repeat(3)} 0.15)`,
+      };
+    },
+    listbox: {
+      padding: '8px 0',
+      margin: '0 10px',
+    },
+    groupLabel: ({ theme }: { theme: Theme }) => ({
+      ...theme.typography.body2,
+      backgroundColor: theme.palette.mode === 'light' ? 'white' : 'black',
+      padding: '16px 12px',
```

Comment:
Seeing all these `16px`, `12px`... wondering if it makes sense to reference spacing properties on the theme or what that be unnecessarily complicated?

Reply:
Let me run it by Nate. Some of the paddings and margins don't correspond a spacing value. It all depends on whether they're intended to be or not.